lms/djangoapps/teams/tests/test_views.py

```diff
@@ -274,7 +311,7 @@ def test_filter_text_search(self):
     @ddt.data(
         (None, 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
         ('name', 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
-        ('open_slots', 200, ['Wind Team', 'Nuclear Team', u'sólar team']),
+        ('open_slots', 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
```

Comment:
Unfortunately though, the ordering for "name" now matches the ordering for "open_slots". Can you do something to make these different?